### DISCUSSION TITLE
UK vs US spelling

### DISCUSSION THREAD

**Discussion by needlesslygrim** [GradienceTeam/Gradience#299] [General]
Should Gradience be in UK or US English, since it is a program using the word colour so much it is probably reasonably important. Would translation be a way to do this?

**Comment by 0xMRTT** [comment]
I can write source in UK English and add a translation for US English ?

**Reply by needlesslygrim** [reply]
Ok then, next time I am looking at the source I will correct any misspellings.

**Comment by 0xMRTT** [comment]
Added us translation in #304, see on weblate

**Reply by needlesslygrim** [reply]
Does that mean I should change the strings in the source code to UK spelling and then use Weblate to "translate" them to US English?

**Reply by 0xMRTT** [reply]
Yes
